admin_guide/scheduling/node_affinity.adoc

```diff
@@ -0,0 +1,288 @@
+[[admin-guide-sched-affinity]]
+= Using Node Affinity
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+_Node affinity_ is a set of rules used by the scheduler to determine where a pod can be placed. The rules are defined using custom xref:../architecture/core_concepts/pods_and_services.adoc#labels[labels on nodes] and label selectors specified in pods. Node affinity allows a *pod* to specify an affinity (or anti-affinity) towards a group of *nodes* it can be placed on.  The node does not have control over the placement. 
+
+For example, you could configure a pod to only run on a node with a specific CPU or in a specific availability zone.
+
+There are two types of node affinity rules: _required_ and _preferred_.
+
+Required rules *must* be met before a pod can be scheduled on a node. Preferred rules specify that, if the rule is met, the scheduler will try to enforce the rules, but does not guarantee enforcement. If a pod has an affinity rule, but there is no node with a matching label, that pod can be scheduled on a node that does not have a label selector. 
+
+[NOTE]
+====
+If labels on a node change at runtime that results in an node affinity rule on a pod no longer being met, the pod will still continue to run on the node.
+====
+
+[[admin-guide-sched-affinity-config]]
+== Configuring Node Affinity
+
+Node affinity is configured through the pod specification file. You can specify a xref:admin-guide-sched-affinity-config-req[required rule], a xref:admin-guide-sched-affinity-config-pref[preferred rule], or both. If you specify both, the node must first meet the required rule, then will attempt to meet the preferred rule.
```

Comment:
s/then will attempt to meet the preferred rule./then _it attempts_ to meet the preferred rule.